### DISCUSSION TITLE
How to build progressive web app?

### DISCUSSION THREAD

**Discussion by ravinder387** [GenieFramework/Genie.jl#468] [General]
how to build pwa app with genie?

**Comment by AbhimanyuAryan** [comment]
@ravinder387 we don't support pwa apps with Genie atm. However, with that said PWA is an interesting area. If more people want this feature, In future we might push this in our Genie Pipeline maybe as a separate plugin. However atm the top priority for us remains improving docs for users

**Reply by ravinder387** [reply]
thanks
